### PR TITLE
Update tutorial simple blueprint

### PR DESCRIPTION
**Pull request by reshnm**
**How to categorize this PR?**

/area documentation
/kind bug
/priority 4

**What this PR does / why we need it**:
The tutorial has some minor bugs in the blueprint example.
The repository context needs to be set in the component descriptor before running the local rendering,
otherwise this step will fail.

**Which issue(s) this PR fixes**:
none

**Special notes for your reviewer**:

**Release note**:

```other operator
```

### CODE REVIEW

**Comment by gardener-robot-ci-3** [comment]
Thank you @reshnm for your contribution. Before I can start building your PR, a member of the organization must set the required label(s) {'reviewed/ok-to-test'}. Once started, you can check the build status in the PR checks section below.

**Review by enrico-kaack-comp — APPROVED** [review]
Please keep in mind that the release notes will be collected for each release, so choose descriptive notes or leave them blank (which is fine in this case).

**Comment by reshnm** [comment]
> Please keep in mind that the release notes will be collected for each release, so choose descriptive notes or leave them blank (which is fine in this case).

Makes sense, I've removed them.

File: docs/tutorials/01-create-simple-blueprint.md [hunk 1]
```diff
@@ -344,7 +353,7 @@ landscaper-cli blueprints push \
 Blueprints are also just resources/artifacts of a Component Descriptor. Therefore, after the Blueprint got uploaded, its reference needs to be added to the Component Descriptor. This is necessary to make sure that all resources of an application are known and stored - and the Blueprint is just one of the resources of an application.
 In addition, Landscaper needs this information to resolve the location of the Blueprint resource.
 
-Note that the repository context as well as the Blueprint resource should be added to the Component Descriptor.
+Note that the Blueprint resource should be added to the Component Descriptor.
```
Comment (schrodit) [inline docs/tutorials/01-create-simple-blueprint.md:356]: I would leave that comment or tell that the repository context should be replaced with the repository context for the own registry

**Review by schrodit — CHANGES_REQUESTED** [review]

File: docs/tutorials/01-create-simple-blueprint.md (hunk 1 above)
Comment (reshnm) [reply docs/tutorials/01-create-simple-blueprint.md]: Ok, I have reverted the change of this sentence.

**Review by schrodit — APPROVED** [review]
/lgtm

**Pull request merged**